Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverUsers/PoweredLightComponent.cs

```diff
@@ -25,17 +29,26 @@ namespace Content.Server.GameObjects.Components.Power.ApcNetComponents.PowerRece
     ///     Component that represents a wall light. It has a light bulb that can be replaced when broken.
     /// </summary>
     [RegisterComponent]
-    public class PoweredLightComponent : Component, IInteractHand, IInteractUsing, IMapInit, ISignalReceiver<bool>, ISignalReceiver<ToggleSignal>
+    public class PoweredLightComponent : Component, IInteractHand, IInteractUsing, IMapInit, ISignalReceiver<bool>, ISignalReceiver<ToggleSignal>, IGhostBooAffected
```

Comment:
I'm kinda suss on tacking on another interface for ghost-specific stuff and feel it should maybe be in a separate component to keep the components a bit more passive but see how Paul feels about it.

Especially considering if you want a light that's not affected by ghosts now isn't possible.

Reply:
I think that new interface is more flexible. You can implement interface using new component or reuse old one. Also saves some components overhead.
Still I can implement something like `GhostFlickerableLightComponent` that implements `IGhostBooAffected`. But sounds like overkill to me.

> Especially considering if you want a light that's not affected by ghosts now isn't possible.

Well, I can create new serialized bool for `PoweredLightComponent`, like `ignoreGhostsBoo`.

Reply:
yeah just do that probably and then downstreams can just toggle it.